lib/v2/ieee/latestdate.js

```diff
@@ -0,0 +1,102 @@
+const got = require('@/utils/got');
+const path = require('path');
+const { art } = require('@/utils/render');
+
+module.exports = async (ctx) => {
+    const punumber = ctx.params.journal;
+    const sortType = ctx.params.sortType || 'vol-only-seq';
+    const hostUrl = 'https://ieeexplore.ieee.org/xpl/mostRecentIssue.jsp?punumber=' + punumber;
+    const headers = {
+        Host: 'ieeexplore.ieee.org',
+        Accept: 'application/json, text/plain, */*',
+        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.45 Safari/537.36',
```

Comment:
https://github.com/DIYgod/RSSHub/blob/a43b67bc1f6d04519c09ac04f3fce4c9e074381a/lib/config.js#L40

`UA` should be configured via env instead of hard-coded.

Reply:
Thanks your attention, I have modified it.
```javascript
const config = require('@/config').value;
const headers = {
    Accept: 'application/json, text/plain, */*',
    'Accept-Encoding': 'gzip, deflate, br',
    'Accept-Language': 'zh-CN,zh;q=0.9',
    Host: host.replace('https://', ''),
    'User-Agent': config.ua,
    Referer: hostUrl,
};
```